pkg/dagger.io/dagger/core/start.cue

```diff
@@ -0,0 +1,96 @@
+package core
```

Comment:
We moved away from the `async` terminology in the original PR (i.e. I changed the name from `core.#AsyncExec` to `core.#Start`), so I feel like it would be out of place now.

I did just realize that I never updated the actual go structs to remove the term `async`, which is maybe where the confusion is coming from. I'll update with that.

Reply:
Yes my bad, I was confuse with this..

Reply:
No worries! I changed the name of the go structs to just be `startTask` and `stopTask` now